src/ir_Whirlpool.cpp

```diff
@@ -38,8 +38,13 @@ using irutils::addModeToString;
 using irutils::addModelToString;
 using irutils::addTempToString;
 using irutils::minsToString;
-using irutils::setBit;
-using irutils::setBits;
+
+#define GETTIME(x) _.x##Hours * 60 + _.x##Mins
```

Comment:
I think the safer way to do this is:
```
#define GETTIME(x) ( _.x##Hours * 60 + _.x##Mins )
```

to protect against macro expansion issues.

**(Required Change)**